src/Workspaces/Core/Portable/Log/FileLogger.cs

```diff
@@ -45,7 +46,7 @@ public FileLogger(IGlobalOptionService optionService)
         }
 
         private static string GetLogFileName()
-            => DateTime.Now.ToString().Replace(' ', '_').Replace('/', '_').Replace(':', '_') + ".log";
+            => DateTime.Now.ToString(CultureInfo.InvariantCulture).Replace(' ', '_').Replace('/', '_').Replace(':', '_') + ".log";
```

Comment:
Wouldn't it be easier to specify a known format that works? `DateTime.Now.ToString("YYYY_MM_DD_hh_mm_ss")` or something?

Reply:
Seems reasonable, I'll make the change next time I am touching this code.